src/baguetteBox.js

```diff
@@ -48,7 +48,10 @@
             afterShow: null,
             afterHide: null,
             onChange: null,
-            overlayBackgroundColor: 'rgba(0,0,0,.8)'
+            overlayBackgroundColor: 'rgba(0,0,0,.8)',
+            closeX: closeX,
+            leftArrow: leftArrow,
+            rightArrow: rightArrow,
```

Comment:
Please add `Icon` suffix to these options:

```suggestion
            rightArrowIcon: rightArrow
```